### PR TITLE
add get user roles consumer

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

File: src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.RightsService.Broker.Consumers
+{
+    public class GetUserRolesConsumer : IConsumer<IGetUserRolesRequest>
+    {
+        private readonly IUserRepository _repository;
+
+        private object GetRoles(IGetUserRolesRequest request)
```
Comment (MariyaSin) [inline src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs:18]: please, do not use var

**Review by MariyaSin — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        public async Task Consume(ConsumeContext<IGetUserRolesRequest> context)
+        {
+            var response = OperationResultWrapper.CreateResponse(GetRoles, context.Message);
+
+            await context.RespondAsync<IOperationResult<IGetUserRolesResponse>>(response);
+        }
+
+
```
Comment (djino97) [inline src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs:54]: Remove empty lines

**Review by djino97 — APPROVED** [review]

File: src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs]: fixed

File: src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/RightsService.Broker/Consumers/GetUserRolesConsumer.cs:18]: fixed

**Pull request merged**